artifacts/arm-trusted-firmware/rk3328/pkg.yaml

```diff
@@ -5,7 +5,7 @@ dependencies:
   - stage: base
 steps:
   - sources:
-      - url: https://github.com/ARM-software/arm-trusted-firmware/archive/lts-v{{ .arm_trusted_firmware_version }}.tar.gz
```

Comment:
the new url's have `lts-v` could you rebase these non-relevant changes

Reply:
oh the PR is still in draft, sorry my bad

Reply:
Would you like these as a separate PR or should I drop them?

Reply:
just rebase, these files doesn;'t need to be updated for nanopi-r4s right?

Reply:
I need to double check if any LTS includes the necessary support for the NanoPi R5S, which is why I initially switched to using commits.

Reply:
@frezbo So I checked again and the latest LTS is `lts-v2.10.4` from [2024-04-26](https://github.com/ARM-software/arm-trusted-firmware/releases/tag/lts-v2.10.4), but the commit that introduces support for the Nano Pi R5S is from the [2024-06-07](https://github.com/ARM-software/arm-trusted-firmware/commit/fe4df8bdae0a5d1d87c67c24fdd03595cd90dd08). This is why I switched to using the commit ref instead of the LTS tag. Is that fine?

Reply:
yes, that's fine then